addon_toolkit/interfaces/citation.py

```diff
@@ -31,7 +31,7 @@ class CitationConfig:
     external_account_id: str | None = None
 
 
-class ItemType(Enum):
+class ItemType(StrEnum):
```

Comment:
I'm curious will this change affect the `Zotero` and `Mendeley API` implementations? If not leave it as you did

Reply:
This has been done to fix filters invokation from frontend